backend/src/prompts/templates/answer-user-question.j2

```diff
@@ -1,47 +1,4 @@
-You are an expert in providing accurate and complete answers to user queries. Your task is twofold:
-
-1. **Generate a detailed answer** to the user's question based on the provided content or context.
-2. **Validate** if the generated answer directly addresses the user's question and is factually accurate.
+You are an expert in providing accurate and complete answers to user queries. Your task is to **Generate a detailed answer** to the user's question.
```

Comment:
Since this is being used by the generalist, can we rename this file to `generalist-answer.j2` and can we mention ESG specialism in this prompt as well?